src/main/java/com/github/tomakehurst/wiremock/common/DefaultNetworkAddressRules.java

```diff
@@ -80,4 +80,12 @@ public boolean isAllowed(String testValue) {
           && deniedHostPatterns.stream().noneMatch(rule -> rule.isIncluded(testValue));
     }
   }
+
+  @Override
+  public boolean isAllowedAll() {
+    return allowed.equals(Set.of(ALL))
```

Comment:
I wonder if we should pull out `Set.of(ALL)` as a constant, we've now got it 5 times... not sure what to call it though, since `ALL` is taken.

Reply:
Thank you for the suggestion. I agree that extracting Set.of(ALL) as a constant would improve the readability of the code.
Regarding the placement of the constant, would you prefer it to be in the `NetworkAddressRange` class or should we keep it within the `DefaultNetworkAddressRules` class?

 I was thinking we could name it `ALL_RANGES = Set.of(ALL);`

Reply:
> I wonder if we should pull out `Set.of(ALL)` as a constant, we've now got it 5 times... not sure what to call it though, since `ALL` is taken.

Thank you for the suggestion. I agree that extracting Set.of(ALL) as a constant would improve the readability of the code.
Regarding the placement of the constant, would you prefer it to be in the NetworkAddressRange class or should we keep it within the DefaultNetworkAddressRules class?

 I was thinking we could name it `ALL_RANGES = Set.of(ALL);`

Reply:
I think this is a good name and I suggest we put it in `NetworkAddressRange`

Reply:
e450d5ecebe8e96912d491a01de0397156345905